src/services/jwt.service.js

```diff
@@ -0,0 +1,52 @@
+import jwt from 'jsonwebtoken';
+
+const generateAccessToken = (user) => {
+  return jwt.sign(user, process.env.JWT_KEY, {
+    expiresIn: '10m',
```

Comment:
The access token expiration time is set to '10m'. Ensure this aligns with your application's security requirements. Typically, access tokens have shorter lifespans.